roles/pulp3/defaults/main.yml

```diff
@@ -26,7 +26,7 @@ pulp_config_database:
     ENGINE: django.db.backends.sqlite3
     NAME: '{{ pulp_var_dir }}/database/pulp.sqlite3'
 
-pulp_config_secret_key: ''
+pulp_config_secret_key: 'secret_key'
```

Comment:
This needs to remain empty. It should be instantiated by the implementing playbook or passed in as a command-line parameter.

Reply:
I changed because the test was failing, so I should set it in the test playbook?